wallet/src/types.rs

```diff
@@ -283,6 +277,27 @@ pub struct SuperBlockNotification {
     pub consolidated_block_hashes: Vec<String>,
 }
 
+#[derive(Clone, Copy, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub enum FeeType {
+    Absolute,
+    Relative,
```

Comment:
This used to be called `weighted`, otherwise sheikah sees an unknown variant error.

```suggestion
    Weighted,
```

Reply:
Gotcha. I'll do a serde rename.